tests/test_crs/valid_smallfile.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  workload:
+    name: smallfile
+    args:
+      clients: 1
+      operation: ["create"]
+      threads: 5
+      file_size: 64
+      files: 100
+      storageclass: rook-ceph-block # Provide if PV is needed
```

Comment:
will fail 2/3 CI envs - minikube/minishift because the `rook-ceph-block` sc is not available.

Reply:
so if PV isn't required we can probably remove it?

Reply:
I've added the ability to run on root filesystem